nbs/test_insert.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
_Commented on notebook `nbs/test_insert.ipynb` Cell 13 Line 1_
```markdown
Test None doesn't change anything. Note: The code below raises ZeroDivisionError: integer division or modulo by zero
```
@pydanny Remove my note if it's now returning `{}` :)